### DISCUSSION TITLE
Getting a fatal while trying to push to remote repo

### DISCUSSION THREAD

**Discussion by dezmanvee** [community/community#49099] [Actions] [answered]
### Select Topic Area

Product Feedback

### Body

I create the simple landing page then I create the GitHub repository, and then on my local PC I set the remote using git remote add origin git@github.com:dezmanvee/fotma-repo.git. Afterwards I add, commit, and try pushing my changes using git push -u origin main. I get the following error:

CLI snapshot of error being received

**Comment by tyler-holm** [comment]
This has happened to me a few times in the past when I created a new repository. The first thing I would do is double check that you don't have a typo in the remote url. Running this command should list the url for origin `git remote show origin`. If the remote url is wrong then you'll have to remove it and re-add it.

If that isn't the problem then you can reset the local repo by going to the projects root directory and running `rm -rf .git && git init`. From there you'll have to redo your git add, git commit, git add remote, git branch, and git push. Everything should work after that.

Hopefully this helps!

**Reply by dezmanvee** [reply]
Thank you for your suggestion. I meticulously checked the remote url and found no error. I also reset my local repo and redid the git workflow up to the _git push -u origin main_, unfortunately, i'm getting the same error.

**Comment by ettaboyle** [comment, marked as answer]
Hi there!

Thanks for sharing and we appreciate your patience. You can check our [GitHub Status Page](https://www.githubstatus.com/) to see for real-time and historical data on system performance + subscribe to issues for updates.

**Current Status:** [Investigating reports of degraded performance](https://www.githubstatus.com/incidents/sr7zptjd0b07)

**Reply by dezmanvee** [reply]
oops!
